### DISCUSSION TITLE
FAIMS with DIA-NN(version1.8)

### DISCUSSION THREAD

**Discussion by grazzilybear** [vdemichev/DiaNN#431] [Q&A] [answered]
Does DIA-NN (1.8) support the FAIMS-DIA with multiple CVs? Should I still need to split the CV into different files?

I have tried to run with the original raw files with direct-DIA. It seems working. But I am just wondering is any issue I need to pay attention to? 
If I have one file with 2CV and one file with 3CV, can I run them together with Directe DIA and MBR?

Thanks

**Comment by vdemichev** [comment, marked as answer]
Hi, please see Technology support here https://github.com/vdemichev/DiaNN#raw-data-formats
Identification might work fine without splitting, but for quantification I am pretty sure it's better to split

**Reply by grazzilybear** [reply]
Thanks. So if I split them into different files. For the final protein groups, how should we roll them up to the proteins groups? Thanks

**Reply by vdemichev** [reply]
DIA-NN performs protein inference and asssociates each detected precursor with a protein group - can use these.

**Reply by vdemichev** [reply]
For quantitation, I would choose a CV value for each precursor, which on average results in higher signal. And perform MaxLFQ using iq or diann R package based on these selected values only.

**Reply by fedefratini** [reply]
Hi all, I'm also intereted in the topic and quite new to Dia-NN, I've run some sample at different CV using single CV run and staggered dia windows or multiple CV run fixed range windows and I'd like to compare results, but I can't find the proper annotation of CV on a column of the output file, if not in the title. So, please, I'd like to receive more explanation to how to performed what you suggest "For quantitation, I would choose a CV value for each precursor, which on average results in higher signal. And perform MaxLFQ using iq or diann R package based on these selected values only." thanks
